### ISSUE TITLE
Associate Force Values ul with its h3 heading with aria-labelledby

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#248]
In https://github.com/phetsims/ohms-law/issues/153#issuecomment-583528576 , I realized our group naming conventions were not so consistent. For radiobuttons with visible headings it might be a good rule of thumb to use `aria-labelledby` to associate the group of radios with their heading. Where there is no visible heading, we could consider not using `aria-labelledby`.

@zepumph, hopefully, it is not trouble to add `aria-labelledby` in this case.

This sim is in RC so referencing https://github.com/phetsims/QA/issues/471

**Comment by zepumph** [comment]
It is easy to do for this one case. GFL publication is highest priority, so I'm happy to do it this way for the publication. I do though want to discuss further about if this should end up in common code. We should talk about it more over in ohms-law issues, as that is lower priority and we should do that in the most maintainable way.

**Comment by terracoda** [comment]
@zepumph, the Force Values group name is working GFL regular.

<img width="1062" alt="Screen Shot 2020-02-10 at 3 02 42 PM" src="https://user-images.githubusercontent.com/1902116/74178639-78ecd580-4c16-11ea-8eb7-f374224dc22d.png">

**Comment by terracoda** [comment]
@zepumph, things are working as the should be on VO. Should I close this issue or do you need to leave it open for any reason?

**Comment by terracoda** [comment]
Leaving open for @zepumph to cherry pick.

**Comment by zepumph** [comment]
This was added to rc.2:

![image](https://user-images.githubusercontent.com/6856943/74963615-7f214580-53be-11ea-8a91-f1b52462e7fd.png)


Closing
